python/cudf/cudf/core/dtypes.py

```diff
@@ -224,14 +224,43 @@ class DecimalDtype(ExtensionDtype):
 
     name = "decimal"
     _metadata = ("precision", "scale")
+    _MAX_PRECISION = np.floor(np.log10(np.iinfo("int64").max))
 
-    def __init__(self, precision, scale):
+    def __init__(self, precision, scale=0):
```

Comment:
similarly, should we consider making this `Decimal64Dtype` now rather than later?

Reply:
Renamed to `Decimal64Dtype`.